common/rust/cctrusted_base/src/tdx/rtmr.rs

```diff
@@ -0,0 +1,26 @@
+use crate::tcg::*;
+use std::collections::HashMap;
+
+#[allow(dead_code)]
+pub struct TdxRTMR {
+    index: u8,
+    digests: HashMap<u8, TcgDigest>,
+}
+
+impl TcgIMR for TdxRTMR {
+    fn max_index(&self) -> u8 {
+        return 3;
+    }
+
+    fn get_index(&self) -> u8 {
+        todo!()
```

Comment:
It seems there are lots of todo. Do we want to merge with this todo?

Would you please help me understand the purpose of this PR?

Reply:
This PR provides a framework to build common trusted base that can be build not only the VM SDK but also container side CCNP server and SDK. With this framework, we can easily build VM SDK on this, e.g.: implement the CC APIs like get_cc_quote, get_cc_measurement, get_cc_eventlog etc hence the todos are the place hold of these APIs to what the framework is capable of.
These APIs are in development now as well and will file PRs soon.